examples/wireshark.py

```diff
@@ -123,7 +130,7 @@ def printResult(result):
     for key in print_order:
         string = key + "  " + str(result[key][0]) + "  "\
                  + str(result[key][1]) + "  " + str(result[key][2])
-        print (string)
+        print(string)
```

Comment:
OK, then `from __future__ import print_function` is missing at the very first line of file to be compatible with Py2/Py3.